sos/policies/distros/redhat.py

```diff
@@ -430,11 +433,26 @@ def upload_sftp(self):
                                                        password=_token)
         raise Exception("Could not retrieve valid or anonymous credentials")
 
+    def check_file_too_big(self, archive):
+        size = os.path.getsize(archive)
+        # Lets check if the size is bigger than the limit.
+        # There's really no need to transform the size to Gb,
+        # so we don't need to call any size converter implemented
+        # in tools.py
+        if len(str(size)) >= 10 and (size >= self._max_size_request):
+            self.ui_log.error(
+                _("Size of archive is bigger than Red Hat Customer Portal "
+                  f"limit for uploads of {self._max_size_human_readable} via "
+                  "sos http upload. \n")
+                  )
```

Comment:
It seems strange to stick `1Gb` in a variable for it only to be used once in an f-string, rather than just making the warning message contain that static value.

Reply:
Sure. I'll change that.